packages/react-examples/src/react/__snapshots__/CommandBar.Basic.Example.tsx.shot

```diff
@@ -827,10 +824,11 @@ exports[`Component Examples renders CommandBar.Basic.Example.tsx correctly 1`] =
             </div>
             <div
               className=
-                  ms-OverflowSet-overflowButton
+                  ms-CommandBar-primaryCommand
```

Comment:
Do you know why this class name change happened?

Reply:
Thanks for catching this! getNativeProps was placed incorrectly in the wrapper div and was not applying the className properly.

Reply:
Nice to see the snapshot tests being useful! :)

Reply:
Hmm original question still stands though--why did the class name change (even after the update to native props location)?

Reply:
I updated the pr and it should be good now. The snapshot change was due to `ButtonGrid` applying {...props} after the classNames and then applying and overwriting them.

The remaining snapshot changes for `VerticalBarChart`,`AreaChart`, `OverflowSet`, and `ReizeGroup` are due to the `FocusZone` removal from `OverflowSet` and id updates: